css/main.css

```diff
@@ -27,3 +27,14 @@ mark {
 .indented-level3 {
     padding-left: 60px;
 }
+
+/* Responsive table of contents (toc) */
+
+/* Hides toc on screen widths smaller than 1300px */
+/* page-nav appears on screens 1300px wide and above */
+@media screen and (min-width: 1299.98px) {
```

Comment:
Just one question, why not just 1300px?

Reply:
I wanted to be consistent with `page-nav.css` which has the following:

```
/* Responsive page navigation */

/* Hides page navigation on screen widths smaller than 1300px */
@media screen and (max-width: 1299.98px) {
    #page-nav {
        border: none;
        overflow: hidden;
        padding: 0;
        width: 0;
    }
}
```